packages/table-react/src/Table.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { TableAnchorRowData, TableRow } from "./TableRow";
+
+export type TableRowData = string[] | TableAnchorRowData;
+
+interface Props {
+    columns: string[];
+    rows: Array<TableRowData>;
+}
+
+export function Table(props: Props) {
+    return (
+        <table className="jkl-table">
+            <thead>
+                <tr className="jkl-table__row">
+                    {props.columns.map((columnValue, i) => (
+                        <td className="jkl-table__heading" key={i}>
+                            {columnValue}
+                        </td>
+                    ))}
+                </tr>
+            </thead>
+            <tbody>
+                {props.rows.map((row, i) => (
```

Comment:
I followed the principle of YAGNI here.

https://reactjs.org/docs/reconciliation.html

> As a last resort, you can pass an item’s index in the array as a key. This can work well if the items are never reordered, but reorders will be slow.

>Reorders can also cause issues with component state when indexes are used as keys. Component instances are updated and reused based on their key. If the key is an index, moving an item changes it. As a result, component state for things like uncontrolled inputs can get mixed up and updated in unexpected ways.

1. We don't have anyone (yet) with really big tables. Reordering a table with 10 rows doesn't really get slow even if reordering is slow.
2.  Only valid children of the table are strings and they cannot have uncontrolled state, so no Heisenbugs.

Imo we should add the ability to pass a row ID when it is needed.

Reply:
I'm very open on changing this btw @piofinn @lfbergee but as far as I can see we don't have any of the problems in the medium article. Any thoughts?

Reply:
Not a problem right now. The question is, do we want to have to remember to fix this when someone decides they need a sortable table with 100 rows?

Reply:
Hopefully we will fix it when it becomes a problem and not before. To cite the grand dad of programming Donald Knuth:

> The real problem is that programmers have spent far too much time worrying about efficiency in the wrong places and at the wrong times; premature optimization is the root of all evil (or at least most of it) in programming.